content/docs/user-guide/project-structure/index.md

```diff
@@ -0,0 +1,18 @@
+# Project Structure
+
+Using `dvc init` in your <abbr>workspace</abbr> will start a <abbr>DVC
+project</abbr>, including the default internal
+[`.dvc/`](/doc/user-guide/project-structure/internal-files) directory. From
+there on, you will create and manage different DVC files and populate the
+<abbr>cache</abbr> as you use DVC and work on your data science experiments.
+
+- `dvc.yaml` files define stages that form the pipeline(s) of a project.
```

Comment:
let's highlight the same structure as in the sidebar here. May be even visualize it somehow?

Reply:
OK, I updated the bullet list to match and look like the nav. I'm not sure what you mean by "visualize it" though.